frontend/website/static/js/leaderboard.js

```diff
@@ -54,11 +75,50 @@ function start() {
         // Hide the loader
         document.getElementById("leaderboard-loading").style.display = "none";
       },
-      function(reason) {
+      function (reason) {
+        console.log("Error: " + reason.result.error.message);
+      }
+    );
+}
+
+function startChallenges() {
+  gapi.client
+    .init({
+      apiKey: apiKey
+    })
```

Comment:
I'm not sure it makes sense to initialize the `gapi` client twice -- we could do so once and pass the "it's initialized promise" into the `startChallenges` and `startLeaderboard` functions.

Reply:
I guess this would involve making only one call to `gapi.load("client"` below and in the callback passed there initializing `gapi` client, and the passing it to these two functions

Reply:
see last commit for this! 2b0bdb34ef258cae40aa2568601d0f2879e9b0b0